README.md

```diff
@@ -22,76 +22,115 @@ developing distributed systems.
 Currently there are no tools that correspond a PlusCal/TLA+ spec with
 an implementation of the spec. PGo is a tool that aims to connect the
 specification with the implementation by generating Go code based on a
-PlusCal specification. PGo enables the translation of a verified
-PlusCal specification of a distributed system algorithm into a
-semantically equivalent Go program.
+specification written in Modular PlusCal. The "Modular" prefix
+comes from the need to distinguish the description of a system from
+the model of its environment, which is needed for model checking.
+PGo enables the translation of a Modular PlusCal description of a
+distributed system to verifiable PlusCal,
+as well as to a semantically equivalent Go program.
 
 ## Current status
 
-Actively under development. PGo supports compilation of most
-uni-process and very simple multiprocess PlusCal algorithms into
-corresponding compilable and runnable Go code.
+Actively under development. PGo supports compilation of all PlusCal
+control flow constructs, but not procedures.
+PGo also supports a vast majority of the value-level TLA+ supported by TLC.
 
 See [`manual.pdf`](https://github.com/UBC-NSS/pgo/blob/master/manual.pdf) in the
```

Comment:
Maybe it's good to mention that the `manual.pdf` is not updated at this moment.

Reply:
Ah, you found the one place I forgot to write that. Adding.